### ISSUE TITLE
[Bug] Hang (of export) while exporting backglass data

### ISSUE THREAD

**Issue by gorgatron1** [syd711/vpin-studio#516]
**Describe the bug**
hang (of export) while exporting backglass data

**To Reproduce**

I had just gone through all my backglass media to make sure DMDs were enabled/disabled, update DMD media, etc.  I exported the backglass data:

```
curl http://pinbot.local:8089/api/v1/export/backglasses/plain > ~/backglasses.csv
```

and I came back later and noticed it was still working on it.  I am pretty sure I was editing tables (importing, upgrading, messing with media) while the export was running.

I looked at the output and it looks like it made it all the way through the export before hanging.

[vpin-studio-logs.zip](https://github.com/user-attachments/files/17267459/vpin-studio-logs.zip)

FWIW exporting back glasses normally takes about 4.5 minutes.

**Expected behavior**

Ideally no hang, but if there was a problem/conflict (edit while exporting) then an error would be ok, though ... that might be hard if it is doing a streaming download and has already generated headers.

Maybe there is nothing to be done -- just don't do that :-)

**Comment by syd711** [comment]
![image](https://github.com/user-attachments/assets/add9fa3c-34a8-4455-9df1-d4dd40c21774)

The fix is is a bit larger and comes with the re-creation of the media cache.
As a result, the export should be faster too, because the assets won't extracted everytime from the backglass anymore.

**Comment by davidkoski** [comment]
Niiice

**Comment by syd711** [comment]
Fixed with 3.7.2.
